### DISCUSSION TITLE
ubuntu에 kustomize 설치시 오류가 발생합니다.

### DISCUSSION THREAD

**Discussion by mistarpeo** [tedilabs/fastcampus-devops#62] [Q&A] [answered]
bash: line 130: /home/sgcho78/kustomize: cannot execute binary file: Exec format error

위와 같은 오류가 발생하여 진행을 하지 못하고 있습니다.

환경은
mac os m1 chip
parallels 에 ubuntu 설치 했습니다.

**Comment by posquit0** [comment, marked as answer]
안녕하세요~
강의 내용에서의 설치는 `arm64` 프로세서 상의 환경을 기준으로 하고 있어 해당 오류가 발생하고 있는 것으로 보입니다.
M1 맥북의 경우 `arm64`가 아닌 `amd64` 프로세서의 운영체제라서 따로 바이너리를 공식 홈페이지에서 설치하여 진행이 필요해보입니다.

다른 도구들도 모두 M1 환경에서는 `amd64` 버전을 찾아 설치해야 되는 점 참고 바랍니다.

**Kustomize**
- https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.4
- 위 링크에서 amd64의 linux 바이너리를 다운로드 받아서 진행하시면 되겠습니다!
- [kustomize_v4.5.4_linux_amd64.tar.gz](https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.5.4/kustomize_v4.5.4_linux_amd64.tar.gz)
